docs/guides/creatingasplitter.md

```diff
@@ -1,17 +1,27 @@
-# Creating a Splitter
+# Creating an Event Splitter
 
-Splitter is a part of TriggerMesh routing solution that has a simple purpose to
-split JSON arrays into multiple CloudEvents.
+An event `Splitter` is part of the TriggerMesh routing solution. It has the simple purpose of splitting JSON arrays into multiple [CloudEvents](https://cloudevents.io/) for further processing.
 
-## Prerequisites
+!!! Info "Prerequisites"
+    You need a working TriggerMesh platform installation. See the [installation steps](installation.md). You can verify that the API is available with the following command:
 
-1. K8s cluster and configured kubectl
-1. [Knative Serving and Eventing Operators](https://knative.dev/docs/admin/install/knative-with-operators/)
-1. [TriggerMesh Function](https://github.com/triggermesh/function)
+    ```console
+       $ kubectl get crd|grep splitter
+       splitters.routing.triggermesh.io                        2021-10-06T09:01:38Z
+    ```
```

Comment:
grep is totally unnecessary here.

```suggestion
    ```console
    $ kubectl get crd splitters.routing.triggermesh.io
    NAME                               CREATED AT
    splitters.routing.triggermesh.io   2021-10-06T09:01:38Z
    ```
```